lib/chai.js

```diff
@@ -4,14 +4,13 @@
  * MIT Licensed
  */
 
-var used = []
-  , exports = module.exports = {};
+var used = [];
 
 /*!
  * Chai version
  */
 
-exports.version = '3.5.0';
+exports.version = require('../package').version;
```

Comment:
Also @meeber - I don't think we should add things like minimum time to review. This stuff happens, it's teething issues, but ultimately no big deal. Nothing egregious has happened - when it does we'll deal with it then!